docs/docs/general/02-Getting Started/write-your-first-automation.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 3
+---
+
+# Write Your First Automation
+
+:::tip
+What you will learn
+
+- Initialize your first automation suite
+- Configure your credentials (workspace id and access token)
+- Understand the generated automation case
+- Run your first automation
+- Troubleshooting any issues
+- Where to go next
+:::
+
+## Quickstart
+
+## Initialization
+
+To create your first automation suite, enter
+
+```shell
+npx askui init
+```
+
+<details>
+  <summary>This will create</summary>
+
+-   a `tsconfig.json`: [a json file specifying the root files and the compiler options required to compile the project](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html),
+-   a folder called `test` which includes:
+    -   `test/my-first-askui-test-suite.test.ts`: an example test with askui,
+    -   a folder called `helper` which contains the `jest.setup.ts` file for setting up the test environment
+    -   a `jest.config.ts` configuration file
+
+</details>
+
+### Configuration
+
+Generate credentials through the [user portal](https://app.v2.askui.com/) (usage is free!). Please see the [documentation](https://docs.askui.com/docs/general/askui%20User%20Portal/signup) on signing up and generating credentials.
```

Comment:
I think it makes sense to put the user portal link into a config that we then refer so that we only have to change it in one place.

Reply:
Not in this PR